src/test/kotlin/org/rust/RsTestBase.kt

```diff
@@ -140,6 +141,12 @@ abstract class RsTestBase : BasePlatformTestCase(), RsTestCase {
         testCargoProjects.setCargoFeatures(packageFeatures, testRootDisposable)
     }
 
+    private fun setupExperimentalFeatures() {
```

Comment:
I was just looking for something like this in TOML tests that need the new local crate index! Great.